internal/policies/mount/mount_test.go

```diff
@@ -94,14 +94,14 @@ func TestApplyPolicy(t *testing.T) {
 
 		/***************************** USER ****************************/
 		// Error cases.
-		"error when user is not found":                                               {objectName: "dont exist", wantErr: true},
-		"error when user has invalid uid":                                            {userReturnedUID: "invalid", wantErr: true},
-		"error when user has invalid gid":                                            {userReturnedGID: "invalid", wantErr: true},
-		"error when userDir has invalid permissions":                                 {readOnlyUsersDir: true, wantErr: true},
-		"error when path already exists as a directory":                              {pathAlreadyExists: true, wantErr: true},
-		"error when cleanup with invalid user":                                       {entries: []string{"no entries"}, objectName: "dont exist", wantErr: true},
-		"error when cleanup with no entries and path already exists as a directory":  {entries: []string{"no entries"}, pathAlreadyExists: true, wantErr: true},
-		"error when cleanup with empty entry and path already exists as a directory": {entries: []string{"entry with no value"}, pathAlreadyExists: true, wantErr: true},
+		"user, error when user is not found":                                               {objectName: "dont exist", wantErr: true},
```

Comment:
I like running all error cases with `TestFoo/error`, I think (we did that for machine and users in other tests) that we can postfix rather with `(user)` or `(machine)` and so do that in the positive cases too.

Reply:
What if we go for something like `error-user` or `error user`? This way, you can still run all the error cases with `Test/error` while still keeping them separated enough to run something like `Test/user` or `Test/error_user`

Reply:
If we want to go that way, we should use the same pattern for every tests. So I suggest that we comply with the other tests for now (knowing that we can then use `.*/error.*user` already).

And we can move all the code base to another pattern if felt necessary.

Reply:
yes, but the test name all contains user and machine, normally